.github/workflows/cmake-macos-arm.yml

```
@@ -109,3 +109,20 @@ jobs:
         cd $GITHUB_WORKSPACE/Application/Tests
         ./run_unix.bash
         echo "::endgroup::"
+    - name: Upload failure artifacts (.csv, .npz, .results, .pv, logs, html, post-link logs)
+      if: failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: macos-arm64-failure-artifacts
+        path: |
```

actionlint (1.7.7)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)